src/Marten.NodaTime/NodaTimeExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Marten.Linq.Parsing;
+using Marten.Services;
+using NodaTime;
+using NodaTime.Serialization.JsonNet;
+using Npgsql;
+
+namespace Marten.NodaTime
+{
+    public static class NodaTimeExtensions
+    {
+        public static void UseNodaTime(this StoreOptions storeOptions)
+        {
+            NpgsqlConnection.GlobalTypeMapper.UseNodaTime();
+
+            storeOptions.Linq.MethodCallParsers.Add(new SimpleNodaTimeEqualsParser());
+            storeOptions.Linq.MethodCallParsers.Add(new SimpleNodaTimeNotEqualsParser());
+
+            var serializer = new JsonNetSerializer();

```

Comment:
This will not work with a custom ISerializer registered. Maybe leave the serialization configuration out? and assume the registered ISerializer can handle NodaTime serialization

Reply:
@mdissel That's valid point, although I do believe, that most of our users are not overriding default serializer. I added additional parameter that gives possibility to opt out from Marten cofniguring serializer from you. I also changed implementation to try to use current serializer (so keeping current setup) instead creating new one.
What do you think?